crate/operator/main.py

```diff
@@ -550,3 +552,48 @@ async def secret_update(
                             id=f"update-{crate_custom_object['metadata']['name']}-{user_spec['name']}",  # noqa
                             timeout=config.BOOTSTRAP_TIMEOUT,
                         )
+
+
+@kopf.on.field(
+    "",
+    "v1",
+    "services",
+    labels={LABEL_PART_OF: "cratedb", LABEL_MANAGED_BY: "crate-operator"},
```

Comment:
Filtering for resources that are managed by the operator ONLY so we don't pick up random other services.